action.yml

```diff
@@ -15,6 +15,10 @@ inputs:
     oidc-audience:
         description: "By default, this is the URL of the GitHub repository owner, such as the organization that owns the repository."
         required: false
+    disable-job-summary:
+        description: "Allows to disable to generation of Job Summaries."
+        default: "false"
```

Comment:
```suggestion
        default: false
```

Reply:
The default input should be as string:
![image](https://github.com/jfrog/setup-jfrog-cli/assets/23456142/7d2d24ae-2b7e-4625-a9a0-80681096e879)
The getBooleanInputs maps the possible string booleans into booleans.

Reply:
In other Actions, GitHub uses boolean defaults: 
https://github.com/actions/setup-java/blob/main/action.yml#L48
https://github.com/actions/setup-go/blob/main/action.yml#L11
https://github.com/actions/setup-node/blob/main/action.yml#L16
https://github.com/actions/setup-python/blob/main/action.yml#L17

I'm also afraid users would provide "true" instead of true by mistake.

Reply:
You're right, i just got a syntax warning on my scheme. but it works. 👍